src/capi/azext_capi/_help.py

```diff
@@ -32,6 +32,8 @@
   - name: --ephemeral-disks -e
     type: string
     short-summary: Use ephemeral disks
+  - name: --external-cloud-provider
+    type: string
```

Comment:
Based on the way we're using it in the templates, I think `EXTERNAL_CLOUD_PROVIDER` should be a boolean value that defaults to False. Then in the templates we substitute "external" for True and "azure" for False.

Reply:
Thanks for the reference, I was assuming we wanted to specify provider